main.cc

```diff
@@ -2127,6 +2126,7 @@ To start the scylla server proper, simply invoke as: scylla server (or just scyl
                 ss.local().drain_on_shutdown().get();
             });
 
+            auth_service_future.get();
```

Comment:
It's a pity, that standard auth service start-shutdow-stop sequence is broken. Can you please introduce `future<> auth_service::ensure_superuser_is_created()` and explicitly call it here (or even better -- few lines above, where `db.invoke_on_all(&replica::database::revert_initial_system_read_concurrency_boost).get();` is called. The `auth_service` itself would need maintain the `promise<> superuser_created` and the `auth_service::start()` would start the needed background fiber to resolve it.